src/Utils/formatters.js

```diff
@@ -0,0 +1,2 @@
+export const showDate = date => `${date.getDate()}/${date.getMonth()}/${date.getFullYear()}`;
```

Comment:
you can do it with  toLocaleDateString. Also the main functionallity of this function is to format a date no to show the date, that's why the name should be formatDate
```
export const formatDate = date => date.toLocaleDateString('en-GB')
```

Reply:
Applied change